src/binary-reader-ir.cc

```diff
@@ -526,6 +526,15 @@ Result BinaryReaderIR::OnFuncType(Index index,
   auto func_type = std::make_unique<FuncType>();
   func_type->sig.param_types.assign(param_types, param_types + param_count);
   func_type->sig.result_types.assign(result_types, result_types + result_count);
+
+  module_->features.simd |=
```

Comment:
I'm don't understand the code/standard well enough to know if the places this condition is tracked is sufficient. The upside I think is that even if we miss one, that is not security bug and easy enough to fix. So this works. Does FuncTypes also cover externrefs etc.?